docs/source/user-guide/ir/types.rst

```diff
@@ -100,15 +100,14 @@ Atomic types
 Pointer Types
 =============
 
-llvmlite presently supports both *Typed Pointers* and *Opaque Pointers*. Typed
-Pointers are currently the default; Opaque Pointers will become the default in
-future, and support for Typed Pointers will be eventually be removed.
+The IR layer presently supports both *Typed Pointers* and *Opaque Pointers*.
+Support for Typed Pointers will be eventually be removed.
```

Comment:
I think there's a typo here (an extra "be" around "eventually").

Reply:
Thanks, now fixed!